build.json

```diff
@@ -6,6 +6,9 @@
     "PGSQL_VERSION": {
       "default": "15"
     },
+    "RMQ_VERSION": {
+      "default": "3.9.13"
```

Comment:
But this version is only for the RMQ from conda. It in principle not able to get the concrete version of RMQ from apt. I think this just bring us unnecessary maintain burden to align the version. 

But I am still fine with the refactoring, which is indeed more clear.

Reply:
But we don't get RMQ via APT in arm64 builds, we download it directly from Github releases and can select which version we want:

https://github.com/aiidalab/aiidalab-docker-stack/blob/3208794c1bf1f6faf4de34c4baed8b3a5622b930/stack/base-with-services/Dockerfile#L41

We *do* install Erlang from apt and don't have control over its version, so we can only choose RMQ versions that are supported by it, that is true. There is a big comment about it in the Dockerfile linked above. I agree that is a problem. I am hoping that before the next version bump we'll have conda packages on arm64, I'll push for that at the end of this month.